packages/nextjs/test/integration/test/server/tracingHttp.test.ts

```diff
@@ -24,16 +30,6 @@ describe('Tracing HTTP', () => {
           },
         },
       },
-      spans: [
-        {
-          description: 'GET http://example.com/',
-          op: 'http.client',
-          status: 'ok',
-          data: {
-            'http.response.status_code': 200,
-          },
-        },
-      ],
```

Comment:
I moved this part of the test to an E2E test. I believe `nock` was interfering with the instrumentation and it wasn't generating spans.